core/src/helpers/parseAuctionURL.ts

```diff
@@ -0,0 +1,12 @@
+import convertNumberTo32Bytes from './convertNumberTo32Bytes';
+
+export function parseAuctionURL(url: string): { collateralType: string; auctionID: string } {
+    const parts = url.split(':', 1);
+    const collateralType = parts[0];
+    const auctionID = convertNumberTo32Bytes(parts[1]);
```

Comment:
- Please rename `auctionID` to https://github.com/sidestream-tech/unified-auctions-ui/blob/e50fa92f3a6e5a7c763f0793076ad55faca4b2f0/core/src/types.ts#L6
- Please don't use `convertNumberTo32Bytes` within here, otherwise this function wouldn't be reusable
- Please validate parts (collateral type via getCollateralConfig and `auctionId` to be number)